x-pack/plugins/actions/server/builtin_action_types/lib/send_email.ts

```diff
@@ -68,23 +160,13 @@ export async function sendEmail(logger: Logger, options: SendEmailOptions): Prom
     };
   }
 
-  let useProxy = !!proxySettings;
-
-  if (host) {
-    if (proxySettings?.proxyBypassHosts && proxySettings?.proxyBypassHosts?.has(host)) {
-      useProxy = false;
-    }
-    if (proxySettings?.proxyOnlyHosts && !proxySettings?.proxyOnlyHosts?.has(host)) {
-      useProxy = false;
-    }
-  }
+  const useProxy = getUseProxy(host, proxySettings);
   let customHostSettings: CustomHostSettings | undefined;
 
   if (service === JSON_TRANSPORT_SERVICE) {
     transportConfig.jsonTransport = true;
     delete transportConfig.auth;
-  } else if (service != null) {
-    transportConfig.service = service;
+    return transportConfig;
   } else {
```

Comment:
There was an `else` block to just set the service if service was defined 

```
else if (service != null) {
    transportConfig.service = service;
}
```

that doesn't look like it's there anymore. Where did it get moved? This was what was using nodemailer's well known services to set the host/port if the service was defined.

Reply:
Yeah, I will put it back in place it was before. Thank you for catching!